src/PasswordResetController.js

```diff
@@ -80,7 +80,6 @@ function (Okta, FormController, FormType, ValidationUtil, FactorUtil, FooterSign
         ];
       }
     },
-    Footer: FooterSignout,
```

Comment:
what you are doing is correct, but it is missing something: there is some functionality that FormController.js was providing with the footer. Take a look at [back](https://github.com/okta/okta-signin-widget/blob/master/src/util/FormController.js#L100) in that file. (that method is used in [BaseLoginRouter](https://github.com/okta/okta-signin-widget/blob/master/src/util/BaseLoginRouter.js#L294))
Maybe what we can do is either

a) add a `addFooter` method to FormController and call that wherever we need
```
function addFooter(Footer, options) {
  this.footer = new Footer ...
  this.add(this.footer)
```
or 
b) just set `this.footer` before adding it to the controller.

I prefer option a, but look into what makes more sense

Reply:
I prefer option a as well. So does it look well now?